examples/playground/src/App.tsx

```diff
@@ -0,0 +1,7 @@
+import DrafFrom from '@openfun/vernajs';
+
+function App() {
+  return <DrafFrom />;
```

Comment:
Should it be something like:
```suggestion
  return <VernaForm />;
```

Reply:
Yep the search and replace did not catch it.